portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,23 +22,47 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
```

Comment:
nit: I can't seem to comment on these lines but on lines 18-19, let's remove the unused List and ArrayList imports since we don't need them anymore